### PR TITLE
Add ability to modify node root volume size + aws stack name labels

### PR DESCRIPTION
**Pull request by lowerzero**
*Issue #, if available:*

By default its 20 Gb which can be too low for some workloads.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by lowerzero** [comment]
Also with multiple nodegroups it is handy to be able select nodes from specific cloudformation stack

File: amazon-eks-nodegroup.yaml [hunk 1]
```diff
@@ -242,11 +248,18 @@ Resources:
...
+          Ebs:
+            VolumeSize: !Ref NodeVolumeSize
+            VolumeType: gp2
+            DeleteOnTermination: true
       UserData:
         Fn::Base64:
           !Sub |
             #!/bin/bash
             set -o xtrace
+            export KUBELET_EXTRA_ARGS="--node-labels aws-stack-name=${AWS::StackName}"
```
Comment (micahhausler) [inline amazon-eks-nodegroup.yaml]: Can you please remove this line?

File: amazon-eks-nodegroup.yaml [hunk 2]
```diff
@@ -84,6 +84,11 @@ Parameters:
     Description: Maximum size of Node Group ASG.
     Default: 3
 
+  NodeVolumeSize:
+    Type: Number
+    Description: Node volume size
+    Default: 20
```
Comment (max-rocket-internet) [inline amazon-eks-nodegroup.yaml]: 20GB is very low to have as a default. For example, GKE uses 100GB.

**Comment by max-rocket-internet** [comment]
> Also with multiple nodegroups it is handy to be able select nodes from specific cloudformation stack

This is a really great idea 👍

Would be useful for when you have workers with extra resources for running specific containers, or workers with fast storage, or for when running on spot instances. We could then use a `nodeSelector`.

@micahhausler how come you wanted to remove the `KUBELET_EXTRA_ARGS` line from the user-data?

File: amazon-eks-nodegroup.yaml (hunk 2 above)
Comment (lowerzero) [reply amazon-eks-nodegroup.yaml]: This is default value from AMI

Comment (max-rocket-internet) [reply amazon-eks-nodegroup.yaml]: But that doesn't mean it should be 20 here. It can be set to any value.

Comment (deiwin) [reply amazon-eks-nodegroup.yaml]: @micahhausler has previously mentioned that they want to stick to 20: https://github.com/awslabs/amazon-eks-ami/issues/13#issuecomment-408267434

Comment (max-rocket-internet) [reply amazon-eks-nodegroup.yaml]: Seems a bit old fashioned but OK. At least with this PR we can change it now.

**Comment by micahhausler** [comment]
Thanks! This is great. 

@max-rocket-internet with #23 you can add labels as a parameter to the CloudFormation stack. See [my comment](https://github.com/awslabs/amazon-eks-ami/issues/22#issuecomment-413992153) in #22, so you can add whatever labels and taints you'd like.

**Pull request merged**